src/Microsoft.Health.Fhir.SqlServer/Features/Search/SqlServerSearchService.cs

```diff
@@ -56,7 +57,8 @@ internal class SqlServerSearchService : SearchService
         private readonly SchemaInformation _schemaInformation;
         private readonly ICompressedRawResourceConverter _compressedRawResourceConverter;
         private readonly RequestContextAccessor<IFhirRequestContext> _requestContextAccessor;
-        private const int _resourceTableColumnCount = 10;
+        private const int _defaultResourceTableFinalSelectColumnCount = 11;
+        private bool? _didWeSearchForSortValue;
```

Comment:
>_didWeSearchForSortValue [](http://example.com/codeflow?start=22&length=24)

Would it make sense to push it inside SqlSearchOptions class?

Reply:
Moved this into `SqlSearchOptions` class